### DISCUSSION TITLE
( ImageMagick, ImageMagick (Install), ImageMagick (Portable) ) release notes link is incorrect

### DISCUSSION THREAD

**Discussion by user8446** [chocolatey-community/chocolatey-packages#1910] [Ideas]
Hi,

Just pointing out that on all 3 packages the release notes link incorrectly points to: https://www.imagemagick.org/script/changelog.php

The correct link is: https://github.com/ImageMagick/Website/blob/main/ChangeLog.md

**Comment by AdmiringWorm** [comment]
The imagemagick packages are not maintained in this repository, so you unfortunately have to contact the maintainer of those packages instead (go to the package details page, and click on `Contact Maintainers` or go to its `Package Source` link).

**Comment by user8446** [comment]
Will do, thank you
